spec/support/rspec_api_documentation.rb

```diff
@@ -1,8 +1,18 @@
 require "rspec_api_documentation"
-require "rspec_api_documentation/dsl"
 
 RspecApiDocumentation.configure do |config|
   config.format = %i[json]
   config.curl_host = "http://localhost:3000"
   config.api_name = "API"
+  config.docs_dir = Rails.root.join("doc")
+  config.curl_headers_to_filter = %w[Host Cookie Origin]
+end
+
+# https://github.com/zipmark/rspec_api_documentation/issues/456
+module RspecApiDocumentation
```

Comment:
Can you try to remove it and add these settings to the `RspecApiDocumentation` config?
```ruby
config.request_headers_to_include = %w[Accept Content-Type Authorization]
config.response_headers_to_include = %w[Content-Type]
config.keep_source_order = true
config.request_body_formatter = :json
```